scripts/ci/setup_hadoop_env.sh

```diff
@@ -42,7 +42,7 @@ while test $# -gt 0; do
     esac
 done
 
-HADOOP_HOME=/tmp/hadoop-${HADOOP_DISTRO}
+HADOOP_HOME=${TOX_CACHE_FOLDER}/hadoop_cache/hadoop-${HADOOP_DISTRO}
```

Comment:
Travis has better internet connection than me. So I don't think it makes so much difference. ~30s / 20min